profile/profile.md

```diff
@@ -0,0 +1,64 @@
+# Yerin An . Jarvis Consulting
+
+I am a junior software developer with Jarvis and graduate from Centennial College with an Advanced Diploma of Software Engineering Technology. Also, I have a bachelor's degree in Architecture. I have always been passionate about new technology and thinking of integrating technology into our daily lives. And it gives me to motivate and learn that can improve myself. I have experience as a back-end developer for around a year, and I worked with REST APIs, data analytics, and application development. I look forward to working in data engineering and getting a chance to contribute to the innovations.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, REST APIs, SpringBoot, Hibernate
+
+**Competent:** JavaScript, HTML/CSS, Python, Docker, Algorithms/Data structures, Jupyter notebook, Pandas
+
+**Familiar:** C#, Vue, Android, React, Angular
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_YerinAn](https://github.com/jarviscanada/jarvis_data_eng_YerinAn)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/linux_sql)]: Developed an internal tool to collect and save both server hardware specifications and on-going Linux resource usage data so that users can know their cluster status better. ⋅	Utilized the docker container to provision the database ⋅	Created bash scripts for monitoring nodes ⋅	Developed SQL queries to store and retrieve data
```

Comment:
do not use bullet points..one pharaoh only. 
⋅	Utilized the docker container to provision the database ⋅	Created bash scripts for monitoring nodes ⋅	Developed SQL queries to store and retrieve data

make sure you mention the following keywords - bash, linux, docker, RDBMS or PostgreSQL or SQL or data analytics,